web/api.go

```diff
@@ -30,14 +29,12 @@ type JSONTag struct {
 // @Success 200 {object} []string
 // @Failure 500 {object} map[string]string
 // @Router /api/tags [get]
-func ApiListTag(client consul.Client) gin.HandlerFunc {
+func ApiListTag(client consul.Client, tagsService services.TagsService) gin.HandlerFunc {
```

Comment:
Is there any reason why the consul client is still passed as parameter?
It is not used within the function, and actually, the tags don't use consul at all now.

Reply:
leftover, thx